LLama/Experimental/Runner/LLamaCpp/LLamaCppRunnerInput.cs

```diff
@@ -0,0 +1,92 @@
+﻿using LLama.Experimental.Common;
+using LLama.Native;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+
+namespace LLama.Experimental.Runner.LLamaCpp
+{
+    /// <summary>
+    /// Input special for <see cref="LLamaCppRunner"/>.
+    /// </summary>
+    public class LLamaCppRunnerInput
```

Comment:
Is this the same as `LLamabatch`?

Reply:
Yes, the same reason with above, I don't want to break the current implementations. Instead, I want to keep all things in `Experimental` namespace.